File: Frontend/DisasterDispatchFrontend/src/app/Models/operation-employee.ts
import { Guid } from "guid-typescript";
import { BaseEntity } from "./base-entity";
import { CustomOperation } from "./custom-operation";
import { AppUser } from "./app-user";

export class OperationEmployee extends BaseEntity {
    customOperation:CustomOperation
    customOperationId:Guid
    appUser:AppUser
    appUserId:string
}

File: Backend/DisasterDispatch.Core/Entities/OperationEmployee.cs
﻿using DisasterDispatch.Core.Entities.Common;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DisasterDispatch.Core.Entities
{
    public class OperationEmployee:BaseEntity
    {
        public CustomOperation CustomOperation{ get; set; }
        public Guid CustomOperationId{ get; set; }
        public AppUser AppUser{ get; set; }
        public string AppUserId{ get; set; }
    }
}
